### PR TITLE
Adding new cask MQTT.fx 1.0.0 (token mqttfx)

### PR DESCRIPTION
**Pull request by ftrossbach**
##### Instructions
- Look for and complete the section relevant to your submission. Delete the others, including these `Instructions`.
- `{{cask_file}}` represents the cask file you’re submitting/editing (if applicable).
- If there’s a checkbox you can’t complete for any reason, that’s OK. Just explain in detail why you weren’t able to do so.
#### Editing an existing cask
- [ ] Commit message includes cask’s name (and new version, if applicable).
- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` left no offenses.
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
### Changes to the core
- [ ] Followed [hacking.md](https://github.com/caskroom/homebrew-cask/blob/master/doc/development/hacking.md).

The current version is 1.1.0, but this uses an installer that requires manual steps. The latest automatically installable version is 1.0.0.

### CODE REVIEW

File: Casks/mqttfx.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'mqttfx' do
+  version '1.0.0'
+  sha256 'ffa6d389c5e39e9ce2613598d0ab9e1c187a374889c21f188dbef7aeda9c5177'
+
+  # Homepage refers to this official download location
```
Comment (vitorgalvao) [inline Casks/mqttfx.rb]: Comment needs to be `# jensd.de/apps/mqttfx was verified as official when first introduced to the cask`.

File: Casks/mqttfx.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  url 'http://www.jensd.de/apps/mqttfx/1.0.0/MQTT.fx-1.0.0.dmg'
```
Comment (vitorgalvao) [inline Casks/mqttfx.rb]: `url "http://www.jensd.de/apps/mqttfx/1.0.0/MQTT.fx-#{version}.dmg"`

**Comment by ftrossbach** [comment]
Comments have been incorporated

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**